### PR TITLE
[Console API] Add basic get status API

### PR DESCRIPTION
**Pull request by lewijacn**
### Description
This change introduces a simple get status API to our Console API for retrieving the status of an OSI migration. This logic currently only returns the status and statusMessage from the OSI `getPipeline` API with the anticipation to add more details like progress in future iterations. 

Also included is some reformatting of the response given by each of the API endpoints to keep a consistent response structure on error.

### Issues Resolved
https://opensearch.atlassian.net/browse/MIGRATIONS-1945

### Testing
Unit testing and manual cloud testing

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py [hunk 1]
```diff
@@ -104,34 +110,54 @@ def test_osi_start_migration_assume_role(self):
...
     def test_osi_delete_migration_assume_role(self):
         response = self.client.post('/orchestrator/osi-delete-migration', data=VALID_ASSUME_ROLE_UPDATE_PAYLOAD,
                                     content_type='application/json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIn('Timestamp', response.data)
+        self.assertIn('timestamp', response.data)
+
+    @patch('botocore.client.BaseClient._make_api_call', new=mock_make_api_call)
+    def test_osi_get_status_migration(self):
+        response = self.client.post('/orchestrator/osi-get-status-migration', data=VALID_UPDATE_PAYLOAD,
```
Comment (sumobrian) [inline TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py:147]: Why is get status a 'post'?

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/urls.py [hunk 2]
```diff
@@ -6,4 +6,5 @@
     path('osi-start-migration', views.osi_start_migration, name='osi-start-migration'),
     path('osi-stop-migration', views.osi_stop_migration, name='osi-stop-migration'),
     path('osi-delete-migration', views.osi_delete_migration, name='osi-delete-migration'),
+    path('osi-get-status-migration', views.osi_get_status_migration, name='osi-get-status-migration'),
```
Comment (sumobrian) [inline TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/urls.py:9]: Do we test these API calls within integ tests? I'm concerned about the assumptions about response structure from OSI.

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py (hunk 1 above)
Comment (lewijacn) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py:147]: I did this for convenience initially and forgot to change this to a GET request with a required URL parameter. Will update this

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/urls.py (hunk 2 above)
Comment (lewijacn) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/urls.py:9]: We currently only have unit tests around this, with no integ tests

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py (hunk 1 above)
Comment (lewijacn) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py]: Change has been added now

**Review by sumobrian — APPROVED** [review]
lgtm

**Pull request merged**